plugins/VstBase/RemoteVstPlugin.cpp

```diff
@@ -1577,7 +1577,7 @@ void RemoteVstPlugin::loadPresetFile( const std::string & _file )
 		else
 		{
 			auto toUIntArray = reinterpret_cast<unsigned int*>(chunk);
-			for (int i = 0; i < pBank->numPrograms; i++ )
+			for (auto i = 0u; i < pBank->numPrograms; i++)
```

Comment:
What's `0u` here? I haven't seen this notation anywhere before.

Reply:
It's a literal suffix, like `f` or `l`, etc. It means the literal has type `unsigned int` rather than `int`. See https://en.cppreference.com/w/cpp/language/integer_literal.

Reply:
Ohh i see.

Reply:
But isn't 0 unsigned anyway?

Reply:
The value `0` is non-negative, but that doesn't preclude it having a signed type, since signed integer types are not restricted to negative values only. An integer literal without any suffixes that fits in an `int` will have type `int`, even if it also fits in `unsigned int`.

Reply:
Okk